File: java/pe/src/test/java/ktn/pe/Pe9Test.java
package ktn.pe;

import static org.junit.Assert.*;
import static org.hamcrest.CoreMatchers.*;

import org.junit.Test;

public class Pe9Test {

    @Test
    public void test() {
        Pe9 pe9 = new Pe9();
        
        assertThat("0 0", pe9.pe9(0), is(0));
        assertThat("12 60", pe9.pe9(12), is(60));
    }

}

File: kt/pe/src/test/kotlin/ktn/pe/Pe9Test.kt
package ktn.pe

import org.junit.Assert.*
import org.hamcrest.CoreMatchers.*

import org.junit.Test

class Pe9Test {

    @Test
    fun test() {
        val pe9 = Pe9()

        assertThat("0 0", pe9.pe9(0), `is`(0))
        assertThat("12 60", pe9.pe9(12), `is`(60))
    }

}
